framework/auth/views.py

```diff
@@ -174,6 +177,12 @@ def confirm_email_get(token, auth=None, **kwargs):
 
     if auth and auth.user and auth.user in (user, user.merged_by):
         if not is_merge:
+            # determine if the user registered through a campaign
```

Comment:
Does this imply that *every* login attempt will send the user to the campaign page now and forever?

Is this the intended behavior if, eg, the user continues to use the OSF after the campaign is officially over?